dev/openchpl_data-model.sql

```diff
@@ -27,6 +27,24 @@ CREATE TYPE openchpl.attestation as enum('Affirmative', 'Negative', 'N/A');
 CREATE TYPE openchpl.validation_message_type as enum('Error', 'Warning');
 CREATE TYPE openchpl.job_status_type as enum('In Progress', 'Complete', 'Error');
 
+CREATE TABLE openchpl.chart_data(
```

Comment:
I'm not sure if I'm being too nitpicky here, but putting the word "chart" in the table is tying this data with the way it's being presented for this task. While we aren't using it for anything else right now, we could in the future. If we wanted to do something with it in Excel or PDF then it's may not be specific to charts anymore. I don't have a specific suggestion for what it should be called, just asking you to think about it a little more. Maybe something like statistics and statistics_type.

Reply:
👍